js/src/ui/DappIcon/dappIcon.js

```diff
@@ -41,7 +41,7 @@ export default class DappIcon extends Component {
         src={
           app.type === 'local'
             ? `${dappsUrl}/${app.id}/${app.iconUrl}`
-            : `${dappsUrl}${app.image}`
+            : `${app.image}`
```

Comment:
`/api/content` is now available on `:8180` as well, so we don't need to go to `:8545`.